WalletWasabi.Gui/Controls/QrCode.cs

```diff
@@ -1,20 +1,42 @@
 using Avalonia;
 using Avalonia.Controls;
 using Avalonia.Media;
+using Avalonia.Media.Imaging;
 using Avalonia.Metadata;
+using Avalonia.Platform;
+using ReactiveUI;
 using System;
+using System.Reactive.Linq;
 
 namespace WalletWasabi.Gui.Controls
 {
 	public class QrCode : Control
 	{
+		private readonly int _matrixPadding;
+		private Size _coercedSize;
+		private double _factor;
```

Comment:
Use properties.

Reply:
If the properties are private why not just fields?